common/source/docs/common-geofencing-landing-page.rst

```diff
@@ -0,0 +1,18 @@
+.. _common-geofencing-landing-page:
+
+==========
+GeoFencing
```

Comment:
It's up to you, but personally I like the simpler word "Fence" rather than "GeoFence".  I think the "Geo" part just makes it slightly less understandable to people.

Reply:
I dont care myself...but to be consistent, we need Tridge to buy in for Plane because I will change to it everywhere...

Reply:
Thinking about it, shouldn't we differentiate a geo-fence from a beacon fence?

Reply:
Since you leave it up to me.....GeoFence